README.md

```diff
@@ -43,114 +43,280 @@
 
 # featurevisor-swift
 
-This repository is a work in progress to port the [Featurevisor](https://featurevisor.com) JavaScript SDK to Swift.
-
-We are not ready yet. Please come back later.
+This repository is a port of the [Featurevisor](https://featurevisor.com) JavaScript SDK to Swift.
 
 ## Installation
 
-With [SPM](https://www.swift.org/package-manager/), add the package under `dependencies` in `Package.swift` file:
+### Swift Package Manager
+
+The [Swift Package Manager](https://swift.org/package-manager/) is a tool for automating the distribution of Swift code and is integrated into the `swift` compiler.
+
+Once you have your Swift package set up, adding Featurevisor as a dependency is as easy as adding it to the `dependencies` value of your `Package.swift`.
+
+```swift
+dependencies: [
+    .package(url: "https://github.com/featurevisor/featurevisor-swift.git", .upToNextMajor(from: "X.Y.Z"))
+]
+```
+
+## Usage
+
+### Setting up Featurevisor SDK
+We would like to be able to set up the Featurevisor SDK instance once and reuse the same instance everywhere.
+
+The SDK can be initialized in two different ways depending on your needs.
+
+### Synchronous
+You can fetch the data file content on your own and just pass it via options.
+
+```swift
+import FeaturevisorSDK
+
+let datafileContent: DatafileContent = ...
+var options: InstanceOptions = .default
+options.datafile = datafileContent
+
+let featurevisor = try createInstance(options: options)
```

Comment:
In the website, I have tried using `f` as the instance variable everywhere.

you don't have to change anything, only mentioning.

Reply:
Let's be consistent across our SDKs as much as possible.